components/collection/IndexRecordTable.tsx

```diff
@@ -62,6 +71,16 @@ export const IndexRecordTable = ({
     fetchPlayerHistory()
   }, [playerID])
 
+  const { payload: packs, isLoading } = query<UserCollection[]>({
+    queryKey: ['packs-from-cards', String(cardID), String(userID), "false"],
```

Comment:
The "false" here isn't necessary since it's not a value that changes, but it also isn't causing a problem